packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -629,6 +631,13 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setCenter(center);
     }, [center]);
 
+    /**
+     * Sets app title.
+     */
+    useEffect(() => {
+        document.title = (useAppTitle && appConfig) ? appConfig?.appSettings?.title : 'MapsIndoors Web'
```

Comment:
This will always write something to the document title. We agreed that we should not do that.

I would imagine some customers would be surprised to see that their app using the Map Template web component suddenly has the title of "MapsIndoors Web".